docs/_docs/install/containers.md

```diff
@@ -6,16 +6,20 @@ toc: true
 
 # Installation: Start Containers
 
-Whether you build or not, the compose command will bring up the application 
-(and download containers provided on Quay.io, previously on Docker Hub, if they aren't in your cache).
+Whether you build or not, the compose command will bring up the application (and
+download containers provided on Quay.io, previously on Docker Hub, if they
+aren't in your cache).
 
 ## What containers are provided?
 
-Singularity Registy Server uses the following images, all provided on Quay.io
+Singularity Registry Server uses the following images, all provided on Quay.io
 (or you can build the registry-specific ones locally):
 
- - [quay.io/vanessa/sregistry]({{ site.registry }}): is the core application image, generated from the Dockerfile in the base of the repository.
- - [quay.io/vanessa/sregistry_nginx]({{ site.registry }}_nginx/): Is the nginx container installed with the nginx upload module, intended for use with speedy uploads. It is generated from the subfolder "nginx" in the repository.
+ - [quay.io/vanessa/sregistry]({{ site.registry }}): is the core application
+   image, generated from the Dockerfile in the base of the repository.
```

Comment:
Just to double check - does this still render as part of the previous bullet?

Reply:
oh yeah, looks ok! https://github.com/singularityhub/sregistry/blob/96626d787a20eac6d9ff110e82ee0574ba0854c8/docs/_docs/install/containers.md